### DISCUSSION TITLE
The usage of ForceChildren

### DISCUSSION THREAD

**Discussion by love-code-yeyixiao** [sandboxie-plus/Sandboxie#4044] [Documents and notes]
A marvelous usage based on this feature of ForceChilden.
We can write the following script in a batch file (.bat).
```
cd SandboxieInstallPath
taskkill /f /im Explorer.exe
start.exe /fcp /box:DefaultBox explorer.exe
```
After running it, we will see that Explorer is restarted as a shell.
It doesn't look different, does it? But try launching a notepad using Run (Win+R).
You'll see that it's sandboxed, but we don't have a "forced process" set for Notepad, which is where the ForceChildern feature comes into play!
As you know, explorer can't act as a shell when it's sandboxed, but when it's in this state, even though explorer itself isn't sandboxed, its direct children (excluding UWP apps started through the DCOM interface) will be added to the sandbox we specified in the command.
That's the wonder of the new feature!

**Comment by e-t-l** [comment]
> try launching a notepad using Run (Win+R)

Does this also work when launching an app from the start menu or from a shortcut?

**Reply by love-code-yeyixiao** [reply]
Yes,all applications that are directly starting by Explorer.exe.
